src/pages/send-tokens/transaction-settings.tsx

```diff
@@ -0,0 +1,53 @@
+import { useAtom } from 'jotai';
+import { Box, Text } from '@stacks/ui';
+import { correctNonceState, localOverrideNonceState } from '@store/accounts/nonce';
+import { currentAccountStxAddressState } from '@store/accounts';
+import { Input, InputGroup } from '@stacks/ui';
+import { useAtomValue } from 'jotai/utils';
+import React, { useEffect } from 'react';
+import BigNumber from 'bignumber.js';
+
+export const TxSettings: React.FC = () => {
+  const MAX_LENGTH_BIG_NUMBER = 16;
+
+  const [localNonceOverride, setLocalNonceOverride] = useAtom(localOverrideNonceState);
```

Comment:
Is it weird to use the word `local` here when it is used in multiple components?

Reply:
Yes. This was local before, but not anymore because of the transaction signing popup. I will change it.